bin/asl-say

```diff
@@ -131,10 +145,11 @@ function usage(){
 	exit 1
 }
 
-while getopts "n:w:" opt; do
+while getopts "n:w:d:" opt; do
 	case ${opt} in
 		n)	NODE="${OPTARG}" ;;
 		w)	WHAT="${OPTARG}" ;;
+		d)	DATE_CMD+=" --date=${OPTARG}" ;;
```

Comment:
1. The usage() wasn't updated to indicate the `-d` switch. Do we want to do that?
2. There probably needs to be some quoting in order to process an OPTARG that has spaces in it. Right now using `-d "15 minutes"` doesn't work. It took me several attempts at changes to get this working, but what seemed to work was
Added `DATE_ARG=` at the top of the file,
 Changed this line to `d)	DATE_ARG=--date=${OPTARG} ;;`,
 Replaced all instances of `$($DATE_CMD ` with `$($DATE_CMD "$DATE_ARG"`.

Reply:
Needed to use `eval` :-)